CONTRIBUTIONS.md

```diff
@@ -1,121 +1,1496 @@
+### Avoid Duplicate Work
 
-Both flags are optional. If you don't specify them, it will run the app against the test file.
-And stop it as soon as it becomes healthy. It will timeout after 10 minutes if it doesn't become healthy.
-If you manually `ctrl+c`, you will have to cleanup the leftover containers.
+Before starting work on an app:
 
-The above command will also do the following:
+1. **Check existing issues**: Someone might already be working on it
+2. **Check existing PRs**: The app might already be in review
+3. **Open an issue or comment**: Let others know you're working on it
+4. **Open a draft PR early**: Even an empty PR signals you're working on it
 
-- Generate the `item.yaml` file
-- Update the contents of the `templates/library/` directory based on the `lib_version` on the `app.yaml` file
-- Update the `lib_version_hash` on the `app.yaml` file
+### Repository Structure
 
-ie. if you just want to generate the above, just run the following command, test file can be any, but must exists:
+Once cloned, you'll see this structure:
 
-```bash
-./.github/scripts/ci.py --app qbittorrent --train community --test-file basic-values.yaml --render-only=true
+```
+.
+├── ix-dev/                    # App definitions (this is where you work!)
+│   ├── community/            # Community-contributed apps
+│   ├── charts/               # Legacy apps being migrated
+│   ├── enterprise/           # Enterprise-tier apps
+│   └── stable/               # Stable, production-ready apps
+├── library/                   # Rendering library (Python modules)
+│   └── 2.x.x/                # Library versions
+├── trains/                    # Auto-generated catalog files (DO NOT EDIT)
+├── docs/                      # Documentation
+├── .github/                   # CI/CD scripts and workflows
+│   └── scripts/
+│       └── ci.py             # Local testing script
+└── README.md
 ```
 
-## Testing on TrueNAS System
+**Important:** You should only modify files under `/ix-dev/` or `/library/` directories. All other files are auto-generated.
 
-There is no easy way currently to test on TrueNAS.
-As long as it works on your local machine, it should work on TrueNAS.
-There are some exceptions for things like devices etc, that might be different.
-In such case, let the reviewer know.
+### Understanding Trains
 
-`questions.yaml` is run through some validation during CI, but it still needs some manual check from the reviewer.
-If you want to test how the rendered compose file will look based on different values entered in the questions.yaml,
-you can enter the values you want in one of the test files and render it.
+Apps are organized into "trains" (categories):
 
-## App structure
+- **community**: Community-contributed apps (most new apps go here)
+- **charts**: Legacy apps being migrated from the old system
+- **stable**: Well-tested, production-ready apps
+- **enterprise**: Enterprise-grade applications
```

Comment:
There is no charts train...

Also the train descriptions are not really correct. 
I'll find the correct descriptions for those.